bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3533,6 +3533,7 @@ void ReplicaImp::tryToSendReqMissingDataMsg(SeqNum seqNumber, bool slowPathOnly,
     send(&reqData, destRep);
     metric_sent_req_for_missing_data_++;
   }
+  source_for_rfmd_pp_ = (source_for_rfmd_pp_ + 1) & config_.numReplicas;
```

Comment:
If I'm not wrong it should be config_.numReplicas-1 (when source_for_rfmd_pp_ = 6  the result will be 7 that is not good for us)